apps/ledger-live-desktop/src/renderer/screens/account/BalanceSummary.tsx

```diff
@@ -105,23 +111,36 @@ export default function AccountBalanceSummary({
       </Box>
 
       <Box px={5} ff="Inter" fontSize={4} color="palette.text.shade80" pt={5}>
-        <Chart
-          magnitude={chartMagnitude}
-          color={chartColor}
-          // TODO we need to make Date non optional in live-common
-          data={history as Data}
-          height={200}
-          tickXScale={range}
-          valueKey={displayCountervalue ? "countervalue" : "value"}
-          renderTickY={
-            discreetMode
-              ? () => ""
-              : displayCountervalue
-              ? renderTickYCounterValue
-              : renderTickYCryptoValue
-          }
-          renderTooltip={renderTooltip}
-        />
+        {account.type === "TokenAccount" && account.token.id === "vechain/vtho" ? (
```

Comment:
We do not want vechain specific code here, sorry, please check this for guidelines : https://developers.ledger.com/docs/coin/ui-ux-guidelines/

Reply:
yeah the fix arriving, @haammar-ledger asked that too